rfcs/text/0000-rfc-process-updates.md

```diff
@@ -65,7 +65,9 @@ Additionally, with the encouraged behavior of submitting draft pull requests pri
 We propose adding the following new tags: 
  
 
-**pre-rfc-draft** : This indicates that the authors are still actively working on the RFC, that the RFC is in a draft state, and readers should **not** comment on the RFC yet. Additionally this state serves to indicate more clearly the backlog of RFCs in development to avoid duplication of effort of overlapping RFCs and to help elicit offers of co-authorship.
+**pre-rfc-draft** : This indicates that the authors are still actively working on the RFC, that the RFC is in a draft state, and readers should **not** comment on the RFC yet.
+ 
```

Comment:
I think all mention of pre-RFC should be dropped.  Any work leading up to an RFC should be outside of the scope of defining a formal document.  Without making it part of the formal process, it is just some way people want to work together to wirte the RFC.  It should not *count* as part of the RFC process and could be anything, even a photograph of a whiteboard, a verbal conversation.

People should be free to collaborate on writing an RFC in want ever way they choose.  This should not be dictated.

If the term pre-RFC is confusion, just ban it, don't make it part of the process.

Reply:
Fair enough :) The "pre" part of the tag has been dropped.

Reply:
Hi @diekhans the intent here is to create a one stop shop for folks to see the backlog of rfcs that are currently being drafted. Folks can always keep whatever private docs or working groups they like if they prefer to work in secret.

Here we merely propose a formal way to optionally announce that the pre-rfc exists and that at the least describes the problem being addressed so that an interested party can contact the author(s) to discuss. Does adding "optional" to the text address your concerns?

Reply:
comment below that didn't end up in the right place: I think proposed RFC is clearer than 'pre-RFC', as it indicates something someone is either working on or planning to work on.

Reply:
Agree with @diekhans that pre-RFC shouldn't be part of the process. Would be good however to have a way of communicating what RFCs people are currently working on at the pre-stage (if they want to)

Reply:
Why doesn't a github draft PR also denote that something is a draft?

Reply:
See [related issue](https://github.com/HumanCellAtlas/dcp-community/issues/94).

Reply:
Closing this thread as I believe it has been addressed, but feel free to re-open if not!